atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/service/TransactionService.java

```diff
@@ -46,9 +46,11 @@ public interface TransactionService extends AutoCloseable, AsyncTransactionServi
      */
     @CheckForNull
     @Timed
+    @Deprecated
     Long get(long startTimestamp);
 
     @Timed
+    @Deprecated
```

Comment:
We should tell people what they should use instead (normally this is done with the `@deprecated` annotation in javadoc).